### PR TITLE
Write first usage section

### PR DESCRIPTION
**Pull request by yahwastaken**
Write a draft of the first usage/api sections.

Removed getChannelsOf exports that did not map to an existing implementation.

To be merged into createStore after review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,325 @@
...
+
+import { createStore, subscribeToChannel } from 'aquedux-client'
+import configureAquedux from './configureAquedux'
+import rootReducer from './reducers'
+
+const configureStore = () => {
+  const store = createStore(rootReducer)
+
+  // The default route served by aquedux-server is $YOUR_HOST/aquedux
+  const endpoint = `${protocol}://${host}:${port}/aquedux`
```
Comment (chabou) [inline README.md]: protocol, host, port are unknown variables.
I think this example should be simpler, no need to separate logic in 2 files imo.

File: README.md [hunk 2]
```diff
@@ -1,38 +1,325 @@
...
-You are good to go!
+And you are good to go! For more help you can check out the example/ directory.
+
+# API
+
+wip
+* [ ] Remove onAquedux
+* [ ] Uniform aquedux-client.aqueduxReducer & aquedux-server.aqueduxReducers
+
+## aquedux-client
```
Comment (chabou) [inline README.md]: It should be moved to `packages/aquedux-client/README.md`

File: README.md [hunk 3]
```diff
@@ -1,38 +1,325 @@
...
+  An action creator to subscribe to an aquedux channnel. When the action is sent other the socket, the server returns a state snapshot of the channel's data. After that, until you unsubscribe from the channel, you will receive and reduce every actions related to it. The channels are described through the aquedux client object method `addChannel`.
+
+  * name: The channel's name. *Required*
+  * id: The channel's id. Used only if you want to handle entities through this channel. Default to `undefined`.
+
+  ### unsubscribeFromChannel(name, id)
+
+  An action creator to unsubscribe from an aquedux channnel. See above.
+
+## aquedux-server
```
Comment (chabou) [inline README.md]: It should be moved to `packages/aquedux-server/README.md`

File: README.md [hunk 4]
```diff
@@ -1,38 +1,325 @@
 # <img src='https://user-images.githubusercontent.com/4137761/37289440-f7a26b68-2609-11e8-8c23-fb8b49c53c90.png' height='60'>
-> Aquedux over the wire
+> Redux over the wire
 
+Aquedux is a Redux enhancer that enables seamless real-time synchronization across many JavaScript clients.
```
[the remark this answers is not in the document]
Comment (yahwastaken) [reply README.md]: I agree 👍

File: README.md [hunk 5]
```diff
@@ -1,38 +1,325 @@
...
+* On Linux: Install it with your system package manager. For instance, if it is apt, with `apt-get install redis-server`
+* On Windows: For windows you have two options, either download it from the redis website, or install it on your `Ubuntu On Windows`.
 
-```sh
-// node server side
-yarn add aquedux-server
-// or browser side
-yarn add aquedux-client
+You don't have to configure anything, just running it.
+You now should have a running instance on the localhost:XXXX port.
```
[the remark this answers is not in the document]
Comment (yahwastaken) [reply README.md]: Better remove it then. I did not want to explain how to configure Redis, as it is out of this section scope imo. Could be explained in a "Going further" section. Or even better, a link to an official redis setup documentation.

File: README.md [hunk 6]
```diff
@@ -1,38 +1,325 @@
...
+* [ ] Remove onAquedux
+* [ ] Uniform aquedux-client.aqueduxReducer & aquedux-server.aqueduxReducers
+
+## aquedux-client
+
+  ### createStore(store, ...enhancers)
+
+  Returns a Redux store and initiate the aquedux client connection. It is a facade over the Redux's `createStore` method. See next method for more information about the option parameter.
+
+  ! This method is a shortcut that do the following job.
```
[the remark this answers is not in the document]
Comment (yahwastaken) [reply README.md]: 👍

File: README.md [hunk 7]
```diff
@@ -1,38 +1,325 @@
...
+> Redux over the wire
 
+Aquedux is a Redux enhancer that enables seamless real-time synchronization across many JavaScript clients.
+
+With Aquedux, you can write your server code with the same logic than your web/native app. There is no need to
+use another library that forces you to add GraphQL or RESTfull endpoints. Everything is a Redux app.
+
+Aquedux shares your App state across every connected instance. It can be a Node.js microservice, a React app, or anything you would imagine.
+
+It makes the writing of client and server app easy. There is no need to add another technical layer. If you know how to use Redux, you know how to use Aquedux.
```
[the remark this answers is not in the document]
Comment (yahwastaken) [reply README.md]: Totally agree. As discussed, I think this section deserves its own article with everything you point out. It should be linked here.

**Pull request merged**